src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -274,7 +273,7 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, fromAddress, onNe
               getAddress(quote.inbound_address),
               // This looks incorrect according to https://dev.thorchain.org/thorchain-dev/concepts/sending-transactions#evm-chains
               // But this is how THORSwap does it, and it actually works - using the actual asset address as "asset" will result in reverts
-              AddressZero,
+              getAddress(zeroAddress),
```

Comment:
likewise here, do we need to `getAddress` here or is `zeroAddress` already an address?

Reply:
576b69ac34